edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/KubernetesModule.cs

```diff
@@ -38,7 +39,8 @@ public KubernetesModule(IModule module, KubernetesConfig config)
             ConfigurationInfo configurationInfo,
             IDictionary<string, EnvVal> env,
             KubernetesConfig settings,
-            ImagePullPolicy imagePullPolicy)
+            ImagePullPolicy imagePullPolicy,
+            KubernetesModuleOwner owner = null)
```

Comment:
Is this parameter truly optional? Why not use Option? Also, did you need to add this to the GetHashCode call below?

Reply:
I ask about it truly being optional b/c I see a lot of checks against it being null later on which makes me think that it is a required property...

Reply:
Fixed